tools/Microsoft.Health.Extensions.BuildTimeCodeGenerator/Sql/CreateOrAlterProcedureVisitor.cs

```diff
@@ -248,6 +255,42 @@ private MemberDeclarationSyntax AddPopulateCommandMethodForTableValuedParameters
                                     .WithNameColon(NameColon(ParameterNameForParameter(p)))).ToArray())));
     }
 
+    private MethodDeclarationSyntax[] AddGetOutputMethod(CreateOrAlterProcedureStatement node)
+    {
+        List<MethodDeclarationSyntax> result = new List<MethodDeclarationSyntax>();
+
+        foreach (ProcedureParameter parameter in node.Parameters)
+        {
+            if (parameter.Modifier != ParameterModifier.Output)
+            {
+                continue;
+            }
+
+            TypeSyntax typeSyntax = DataTypeReferenceToClrType(parameter.DataType, parameter.Value != null);
+            result.Add(
+                MethodDeclaration(
+                    typeSyntax,
+                    Identifier($"Get{ParameterNameForParameter(parameter)}FromOutput"))
```

Comment:
If I am reading this correctly, a user would do something like the following, correct?
```csharp
long result = VLatest.Output1.GetResultFromOutput(command);
```
If a user has multiple outputs, I think we should probably return them all in one call, perhaps through a tuple if there is more than one:
```csharp
// One output parameter
long result = VLatest.Output1.GetOutput(command);

// n output parameters
(long result1, long result2) = VLatest.Output1.GetOutput(command);
```

Reply:
Good idea, updated. Thanks